xapian-applications/omega/templates/clicklog

```diff
@@ -0,0 +1,3 @@
+$httpheader{Status,302 Found}
+$httpheader{Location,$cgi{URL}}
```

Comment:
As currently written, this would allow an attacker to create a URL on the domain hosting omega which forwards to a URL of their choosing.  I wonder if that's problematic - we need to be confident it isn't if we're going to add this as a template installed by default.

If we passed through the `DB` parameter then the `URL` parameter isn't actually needed since the URL can be looked up using the `DOCID` parameter: `$field{url,$cgi{DOCID}}` (though that does mean a little extra work as it has to re-fetch and re-parse the document data).